textwiser/options.py

```diff
@@ -433,16 +433,22 @@ class UMAP(_ArgBase):
         ----------
         n_components: int
             The number of topics to use.
+        random_state: Optional[int]
+            The random number seed. Use ``None`` to get non-reproducible results.
```

Comment:
is this a tongue-in-cheek comment? :D Use none to  get non-repr results.. is this ever desirable? may be?

Reply:
I tend to use these non-random options when testing, to quickly try a handful runs and see the range of values I get. But you can also very easily do the same by picking different seeds, so I can be persuaded to remove the optionality.

Reply:
I see. Good to know. From a "documentation" point of view, it was not clear to why would the user want reproducible results. May be re-word? None returns non-deterministic results?